src/actions/guides/database/validating-saving.cr

```diff
@@ -112,6 +112,21 @@ class Guides::Database::ValidatingSaving < GuideAction
     > [save without a params object](#saving-without-a-params-object), for example, in your specs, or in
     > a seeds file.
 
+    ### Bulk updating
+
+    Bulk updating is when you update one or more columns on more than one record at a time.
+    This is a much faster procedure than iterating over each record to update individually.
+
+    ```crystal
+    # Gather all users that are not active
```

Comment:
```suggestion
    # Query for all users that are inactive
```

Changed a bit since "Gather" seemed to me like it was actually getting the uses, butmreally it's just making a query